CHANGELOG.md

```diff
@@ -13,7 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - Add `activeSpeakerPolicy` and `videoUplinkBandwidthPolicy` to `MeetingProvider` props.
-
+- Add `createMeetingResponse` to `VoiceFocusProvider` props.
```

Comment:
What you really did is adding Amazon Chime Echo Reduction, right?

Reply:
It's fine for a CHANGELOG entry to be factual, but it should also be useful.

```
- Allow builders to supply the response from a `CreateMeeting` or `CreateMeetingWithAttendees`
  call when adding a `VoiceFocusProvider` to the component tree. This enables optional features
  like Amazon Chime Echo Reduction to be added to devices when turning on Amazon Chime
  Voice Focus.
```

Reply:
Thanks. updated.